.github/actions/common/constants.py

```diff
@@ -10,7 +10,8 @@ class EventType(Enum):
     'public_linux_debian_10_arm_release',
     'public_linux_fedora_29_x86_64_release',
     'public_linux_ubuntu_20_04_x86_64_release',
-    'public_linux_ubuntu_22_04_arm64_release',
+    'public_linux_ubuntu_22_04_arm64_cross_compilation_release',
```

Comment:
Let's keep it like that (to be aligned with internal configuration)
```suggestion
    'public_linux_ubuntu_22_04_arm64_release',
```